src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -553,6 +554,15 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
             write_to_disk(state_data, output_file_path)
         logger.info(f"Finished writing {observer} files for {state_name}.")
 
+    # Copy over metadata files and subset proportions file
+    metadata_proportions = pd.read_csv(usa_results_dir / "metadata_proportions.csv")
+    state_proportions = metadata_proportions.loc[metadata_proportions["state"] == state]
+    state_proportions.to_csv(state_dir / "metadata_proportions.csv", index=False)
+    # Copy over metadata file
+    metadata_path = usa_results_dir / "metadata.yaml"
+    state_metadata_path = state_dir / "metadata.yaml"
```

Comment:
Not important for this PR, but we may want to include the location of the data in the metadata file (i.e. "USA" for the US data and "RI" for the Rhode Island data.